client/cmd/dexc/main.go

```diff
@@ -84,6 +68,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Catch interrupt signal (e.g. ctrl+c), prompting to shutdown if the user
+	// is logged in, and there are active orders or matches.
+	killChan := make(chan os.Signal)
+	signal.Notify(killChan, os.Interrupt)
+	go func() {
+		for range killChan {
+			if clientCore.PromptShutdown() {
```

Comment:
With an unbuffered `killChan`, if `clientCore.PromptShutdown` is running there is no receiver, so `signal.Notify` does not send repeated notifications.